docs/README.md

```diff
@@ -111,7 +112,7 @@ $ bundle exec jekyll serve --watch
 $ PRODUCTION=1 bundle exec jekyll build
 ```
 
-## API Docs (Scaladoc, Javadoc, Sphinx, roxygen2, MkDocs)
+## Generating individual API Docs (Scaladoc, Javadoc, Sphinx, roxygen2, MkDocs)
```

Comment:
APIs are "Scala", "Java", "Python", "R". roxygen2, mkdocs, sphinx are not APIs

Reply:
Also, I'm confused, weren't the sections above already about generating individual API docs?